### ISSUE TITLE
Command-W doesn't close the plugin installer

### ISSUE THREAD

**Issue by psobolewskiPhD** [napari/napari-plugin-manager#38]
This may be related to https://github.com/napari/napari-plugin-manager/issues/4
but it is an independent issue. 
It should be possible using conventional shortcuts to close the plugin installer window and resume using napari.
Ideally if a plugin is still installing, a dialog would ask for confirmation whether to close and kill the install or cancel.

**Comment by jaimergp** [comment]
Yes, agreed. As you say, the challenge here is what to do when `conda` is still installing things. "Killing the install" is sometimes not feasible depending on which part of the transaction you hit. We would need to let the plugin manager open the installer task queue separately from the dialog. Not sure how much work this is; cc @goanpeca.

**Comment by psobolewskiPhD** [comment]
If Command-W was blocked until install is finished with a warning that would also be fine.

**Comment by psobolewskiPhD** [comment]
Aborting installs is probably a different issue.

**Comment by goanpeca** [comment]
>We would need to let the plugin manager open the installer task queue separately from the dialog. Not sure how much work this is; c

I should be able to get something working within a day. Question @psobolewskiPhD is this `high priotity` in general or thinking in terms of the upcoming release?

**Comment by psobolewskiPhD** [comment]
I just checked, this doesn't work in 0.4.19 either, so this is not a regression. As such, I think this is important, but less urgent regarding 0.5.0
